guides/spin/app/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: single
+title:  "Applications"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Overview
+
+Once you have `spin` [installed and configured](/guides/spin/cli), you can use it to start
+managing your Spinnaker applications.
+
+`spin` can manage the whole lifecycle of your application:
```

Comment:
"Once you have `spin`  [installed and configured](/guides/spin/cli), you can use it to manage you application's lifecycle."

Reply:
Thanks for not incorporating my typo!